Automattic/wp-calypso
# Stats: Adding more banner events

Related to #79339

## Proposed Changes

* adding tracking events for banner dismissal
* adding tracking events for viewing a success banner after purchasing stats

## Testing Instructions



* go to the live branch and trigger the events
* go to the tracking tool and check if an event has been triggered

## Pre-merge Checklist
